src/index.html

```diff
@@ -7,8 +7,38 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
```

Comment:
don't forget about code style, you can check it in (ckecklist)[https://github.com/mate-academy/layout_moyo-header/blob/master/checklist.md]
![image](https://github.com/mate-academy/layout_moyo-header/assets/91411692/ad9e5165-07e0-4af5-b198-21baef38cf90)